bootstrap/kubeadm/api/v1beta1/kubeadmconfig_webhook_test.go

```diff
@@ -337,56 +342,56 @@ func TestKubeadmConfigValidate(t *testing.T) {
 		},
 		"format is Ignition, experimental retry join is set": {
```

Comment:
Can we readd this test?

Reply:
Are you looking at the latest state of the PR? This unit test got re-added already
(except if I"m missing something somehow :D)

Reply:
This should already be re-added. https://github.com/kubernetes-sigs/cluster-api/blob/dd1e6bec00fbf45ea516ea79adb1a92b2fcbb479/bootstrap/kubeadm/internal/webhooks/kubeadmconfig_test.go#L343-L354

Reply:
Damn, local copy seems to be outdated :-(